src/components/checkbox/checkbox.ts

```diff
@@ -231,8 +241,10 @@ export default class SlCheckbox extends ShoelaceElement implements ShoelaceFormC
             : ''}
         </span>
 
-        <div part="label" class="checkbox__label">
-          <slot></slot>
+        <div class="checkbox__label-description-container">
+          <slot part="label" class="checkbox__label"></slot>
+          <div class="checkbox__description-block"></div>
```

Comment:
this is to put space between the label and description. it's _not ideal_, but this section has a line-height in order for it to lineup with the checkbox/radio which does not allow for a margin 😭

Reply:
Yeah, styling checkboxes has always been a PITA